internal/battery/battery.go

```diff
@@ -0,0 +1,172 @@
+package battery
+
+import (
+	"bufio"
+	"fmt"
+	"gobar/internal/blockutils"
+	"gobar/internal/log"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"time"
+)
+
+type bat rune
+
+// Enum of battery icons
+const (
+	BatFull     bat = '\uf240'
+	Bat75       bat = BatFull + 1
+	Bat50       bat = BatFull + 2
+	Bat25       bat = BatFull + 3
+	BatAlert    bat = BatFull + 4
+	BatCharging bat = '\uf0e7'
+)
+
+func getACState(dir string) bool {
+	f, err := os.Open(dir + "/online")
+	if err != nil {
+		log.FileLog(err)
+		return false
+	}
+	defer f.Close()
+
+	sc := bufio.NewScanner(f)
+	sc.Scan()
```

Comment:
do you need a scanner if there's just one line?
or are you out to read just the first line of a multi-line file?